apis/v1alpha2/tlsroute_types.go

```diff
@@ -128,7 +128,9 @@ type TLSRouteRule struct {
 	Matches []TLSRouteMatch `json:"matches,omitempty"`
 
 	// BackendRefs defines the backend(s) where matching requests should be
-	// sent.
+	// sent. If unspecified or invalid, the rule performs no forwarding; if
+	// no filters are specified that would result in a response being sent,
+	// a HTTP 503 status code is returned.
```

Comment:
As above with TCPRoute, it's not always given that TLSRoutes are using HTTP, so maybe something like this:
```suggestion
	// the underlying implementation should actively reject connection attempts to this backend,
	// either by sending a HTTP 503 status code, or closing the connection for non-HTTP TLSRoutes.
```